cross/arm-none-eabi-gdb/Portfile

```diff
@@ -14,3 +14,22 @@ configure.args-append \
 
 # We don't actually violate the tree.
 destroot.violate_mtree  no
+
+# Copied from devel/boost187
+set pythons_versions {2.7 3.9 3.10 3.11 3.12 3.13}
```

Comment:
please do not add EOL versions like 2. 7 (and probably 3.9, which will reach that status in October).